v0/destinations/mp/transform.js

```diff
@@ -169,7 +169,6 @@ function getTransformedJSON(message, mappingJson, useNewMapping) {
     ["traits", "context.traits"],
     MP_IDENTIFY_EXCLUSION_LIST
   );
-  rawPayload = removeUndefinedAndNullValues(rawPayload);
```

Comment:
Can we just keep `remove Undefined values` as it would result in errors

Reply:
Sure. Updating

Reply:
@yashasvibajpai as we are removing undefined values just before encoding we can skip this check here as it i a redundant piece of code here

Reply:
`  const encodedData = Buffer.from(
    JSON.stringify(removeUndefinedValues(parameters))
  ).toString("base64");`
  already checking here, so removing the check as discussed.